charts/templates/coordinator.yaml

```
@@ -23,7 +23,7 @@ spec:
         app.kubernetes.io/component: coordinator
         app.kubernetes.io/created-by: {{ .Values.global.createdBy }}
         app.kubernetes.io/name: coordinator
-        app.kubernetes.io/part-of: marblerun
+        app.kubernetes.io/part-of: {{ .Release.Namespace }}
```

```suggestion
        app.kubernetes.io/part-of: marblerun
```
These should also just be labeled `marblerun`

---

Thanks again, fixed in the latest fixup commit.